presto-cli-rpm/pom.xml

```diff
@@ -0,0 +1,165 @@
+
+                    <packages>
+                        <package>
+                            <name>presto-cli-rpm</name>
+                            <nameOverride>presto-cli-rpm-${project.version}.x86_64.rpm</nameOverride>
+                            <version>${project.version}</version>
+                            <release>1</release>
+
+                            <group>Applications/Databases</group>
+                            <description>Presto CLI RPM Package.</description>
+                            <architecture>x86_64</architecture>
+
+                            <links>
+                                <link>
+                                    <path>/usr/local/bin/presto-cli</path>
+                                    <target>/usr/lib/presto-cli/presto-cli-${project.version}-executable.jar</target>
+                                </link>
+                            </links>
+
+                            <rules>
+                                <rule>
+                                    <destination>/usr/lib/presto-cli</destination>
+                                    <base>${cli.tar.package}</base>
+                                    <!-- make sure jar is executable -->
+                                    <fileMode>0755</fileMode>
+                                    <includes>
+                                        <include>presto-cli-${project.version}-executable.jar</include>
+                                    </includes>
+                                </rule>
+                            </rules>
+                        </package>
+                    </packages>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>enforce-file-size</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireFilesSize>
+                                    <!-- CLI RPM shouldn't be too large -->
+                                    <maxsize>31457280</maxsize>
+                                    <files>
+                                        <file>${project.build.directory}/${project.build.finalName}.x86_64.rpm</file>
+                                    </files>
+                                </requireFilesSize>
+                            </rules>
+                            <fail>true</fail>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>ci</id>
```

Comment:
The IT is added only to CI profile to mirror the `presto-server-rpm` module. The tests currently take around 10s from start to finish.

Reply:
Let's remove this so that it always runs. The reason `presto-server-rpm` has it was due to how the CI used to work when we ran with Travis CI, so we should be able to remove it from there as well.